runtime/amplitude/Cargo.toml

```diff
@@ -25,6 +25,26 @@ smallvec = "1.9.0"
 # Local
 runtime-common = {path = "../common", default-features = false}
 
+# Custom libraries for Spacewalk
+currency = { git = "https://github.com/pendulum-chain/spacewalk", default-features = false, branch = "dev/milestone-3" }
```

Comment:
It is quite risky that we refer to Spacewalk only via a branch name although this branch is still WIP and changing.

Maybe better to add the commit hash (?)

Reply:
I now specified the latest hash of the dev/milestone-3 branch